path.go

```diff
@@ -339,3 +360,11 @@ func GetTrimmedParam(param string) string {
 
 	return param[start:end]
 }
+
+// RemoveEscapeChar remove escape charaters
+func RemoveEscapeChar(word string) string {
+	if -1 != strings.IndexByte(word, escapeChar) {
```

Comment:
@ReneWerner87 , just asking: don't you think that `strings.IndexByte(word, escapeChar) != -1` is a bit clearer and in sync with "the best practices"?)

Reply:
Yes, I can do that too, I am used to the yoda notation from other programming languages, should also bring only advantages here